structure.Rmd

```diff
@@ -355,20 +355,22 @@ In both cases we see two active libraries, consulted in this order:
+This setup is typical on Windows, but is something you usually need to opt into on macOS and Linux[^structure-3].
 With this setup, add-on packages installed from CRAN (or elsewhere) or under local development are kept in the user library.
 Above, the macOS system is used as a primary development machine and has many packages here (\~1000), whereas the Windows system is only used occasionally and is much more spartan.
-The core set of base and recommended packages that ship with R live in the system-level library and are the same on macOS and Windows.
+The core set of base and recommended packages that ship with R live in the system-level library and are the same on all operating systems.
 This separation appeals to many developers and makes it easy to, for example, clean out your add-on packages without disturbing your base R installation.
 
-[^structure-3]: For more details, see the [Maintaining R section](https://whattheyforgot.org/maintaining-r.html#how-to-transfer-your-library-when-updating-r) in *What They Forgot To Teach You About R*.
+[^structure-3]: For more details, see the [Maintaining R section](https://whattheyforgot.org/maintaining-r.html#how-to-transfer-your-library-when-updating-r) in *What They Forgot To Teach You About R*, [Managing Libraries](https://cran.r-project.org/doc/manuals/r-release/R-admin.html#Managing-libraries) in *R Installation and Administration* and the R help files for `Startup` and `.libPaths()`.
 
-If you're on macOS and only see one library, there is no urgent need to change anything.
+If you're on macOS or Linux and only see one library, there is no urgent need to change anything.
 But next time you upgrade R, consider creating a user-level library.
-By default, R looks for a user library found at the path stored in the environment variable `R_LIBS_USER`, which itself defaults to `~/Library/R/m/x.y/library,` (where `m` is a concise description of your CPU architecture, and `x.y` is the R version).
+By default, R looks for a user library found at the path stored in the environment variable `R_LIBS_USER`, which itself defaults to `~/Library/R/m/x.y/library,` on macOS, and `~/R/m-library/x.y` on Linux (where `m` is a concise description of your CPU architecture, and `x.y` is the R version).
+You can see this path with `Sys.getenv("R_LIBS_USER")`.
+These directories do not exist by default, and the use of them must be enabled by creating the directory.
 When you install a new version of R, and prior to installing any add-on packages, use `dir.create(Sys.getenv("R_LIBS_USER"), recursive = TRUE)` to set up a user library.
 Now you will have the library setup seen above.
-Alternatively, you could setup a user library elsewhere and tell R about that by setting the `R_LIBS_USER` environment variable in `.Renviron`.
+Alternatively, you could setup a user library elsewhere and tell R about that by setting the `R_LIBS_USER` environment variable in the `.Renviron` in your `HOME` directory.
```

Comment:
Let's not get into where `.Renviron` is looked for, because technically it's not quite this simple. We can just remind folks to use `usethis::edit_r_environ()` to open it for editing.

Reply:
Ah, great idea.